### PR TITLE
[43479] There is no empty status in the Project Select when search criteria is not met

### PR DESCRIPTION
**Pull request by HDinger**
Show "no results" message for project include and global project select.

https://community.openproject.org/projects/openproject/work_packages/43479/activity

### CODE REVIEW

File: frontend/src/app/shared/components/autocompleter/project-menu-autocomplete/project-menu-autocomplete.template.html [hunk 1]
```diff
@@ -47,6 +47,7 @@
 
       <ng-container *ngIf="(loading$ | async) === false; else loadingTemplate">
         <ul
+          *ngIf="(projects$ | async)?.length > 0; else noResultsTemplate"
           op-project-list
           [projects]="projects$ | async"
```
Comment (Kharonus) [inline frontend/src/app/shared/components/autocompleter/project-menu-autocomplete/project-menu-autocomplete.template.html]: can you asign the subscription result to a variable in the ngIf, so you don't have to use async pipe twice?

Comment (Kharonus) [reply frontend/src/app/shared/components/autocompleter/project-menu-autocomplete/project-menu-autocomplete.template.html]: same in the other places

File: config/locales/js-en.yml [hunk 2]
```diff
@@ -1287,6 +1287,7 @@ en:
         include_all_selected: 'Include all sub-projects is currently selected.'
         current_project: 'This is the current project you are in.'
         does_not_match_search: 'Project does not match the search criteria.'
+      no_results: 'No project matching your search criteria'
```
Comment (Kharonus) [inline config/locales/js-en.yml]: I'd rather upgrade this group of words to a sentence: `No project matches your search criteria.`

IMHO it looks weird when there is no full stop in this place.

**Review by Kharonus — CHANGES_REQUESTED** [review]

File: config/locales/js-en.yml (hunk 2 above)
Comment (HDinger) [reply config/locales/js-en.yml]: @marcalcobe Any thoughts on that?

Comment (marcalcobe) [reply config/locales/js-en.yml]: @HDinger and @Kharonus looks good to me

**Review by Kharonus — APPROVED** [review]
LGTM, no further issues,

**Pull request merged**